config.gradle

```diff
@@ -0,0 +1,7 @@
+ext {
+    releaseConfig = [
+            "group"      : "com.infinum.libname",
+            "version"    : "0.0.1",
+            "versionCode": 0 * 10000 + 0 * 100 + 1
```

Comment:
Do we need version code in the libraries?

Reply:
Sry, I've missed this one. Not sure if we need it. I know it's present on the libraries I've encountered so I suppose it is indeed needed.

Reply:
I'm pretty sure it isn't needed, and the most popular Android libraries such as Timber, Room or Glide don't have it.
What are the examples of libraries that have version code defined?

Reply:
Then I'll eliminate it :) . Sentinel, JsonApiX, Compexify.